token.sh

```diff
@@ -14,22 +13,37 @@ API_VERSION=v3
 API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
 AUTH_HEADER="Authorization: token ${ACCESS_TOKEN}"
 
-REPO_URL=${REPO_URL:-${URI}}
+if [[ ${RUNNER_SCOPE} == "repo" ]]; then
+  [[ -z ${REPO_URL} ]] && ( echo "REPO_URL required for repo runners"; exit 1 )
```

Comment:
this already exists in `entrypoint.sh` and shouldnt be a concern in both places (keep it DRY)